### DISCUSSION TITLE
Please help, I get many errors in the ServeCommand.php file after installation

### DISCUSSION THREAD

**Discussion by tnd-dev99** [qruto/laravel-wave#64] [Q&A]
Undefined array key 1

  at vendor\laravel\framework\src\Illuminate\Foundation\Console\ServeCommand.php:359
    355▕         $line = str_replace('  ', ' ', $line);
    356▕
    357▕         preg_match($regex, $line, $matches);
    358▕
  ➜ 359▕         return Carbon::createFromFormat('D M d H:i:s Y', $matches[1]);
    360▕     }
    361▕
    362▕     /**
    363▕      * Get the request port from the given PHP server output.

  1   vendor\laravel\framework\src\Illuminate\Foundation\Console\ServeCommand.php:359
      Illuminate\Foundation\Bootstrap\HandleExceptions::Illuminate\Foundation\Bootstrap\{closure}("Undefined array key 1", "D:\TND\4xs-laravel\vendor\laravel\framework\src\Illuminate\Foundation\Console\ServeCommand.php")

  2   vendor\laravel\framework\src\Illuminate\Foundation\Console\ServeCommand.php:294
      Illuminate\Foundation\Console\ServeCommand::getDateFromLine("[Thu Apr 11 15:56:16 2024] 127.0.0.1:55891 Accepted")
      
      
I tried editing line 359 and got this error.

  Call to a member function format() on null

  at vendor\laravel\framework\src\Illuminate\Foundation\Console\ServeCommand.php:313
    309▕                     }
    310▕
    311▕                     [$startDate, $file] = $this->requestsPool[$requestPort];
    312▕
  ➜ 313▕                     $formattedStartedAt = $startDate->format('Y-m-d H:i:s');
    314▕
    315▕                     unset($this->requestsPool[$requestPort]);
    316▕
    317▕                     [$date, $time] = explode(' ', $formattedStartedAt);

  1   vendor\laravel\framework\src\Illuminate\Collections\Traits\EnumeratesValues.php:236
      Illuminate\Foundation\Console\ServeCommand::Illuminate\Foundation\Console\{closure}("[Thu Apr 11 16:07:48 2024] 127.0.0.1:56057 Closing")

  2   vendor\laravel\framework\src\Illuminate\Foundation\Console\ServeCommand.php:276
      Illuminate\Support\Collection::each(Object(Closure))
      
 

Please help me

**Comment by slavarazum** [comment]
I'm unsure if it's related to Wave. Please thoroughly debug and create an issue with more details if it is related.
